src/olympia/addons/indexers.py

```diff
@@ -77,6 +77,7 @@ def get_mapping(cls):
                     'hotness': {'type': 'double'},
                     'icon_type': {'type': 'string', 'index': 'no'},
                     'is_disabled': {'type': 'boolean'},
+                    'is_experimental': {'type': 'boolean'},
```

Comment:
Yeah and actually you're searching on it - in the legacy api search view. Carry on.